README.md

```diff
@@ -1,5 +1,81 @@
 # PayPal iOS SDK
 
-![GitHub Actions Tests](https://github.com/paypal/iOS-SDK/workflows/Tests/badge.svg)
+Welcome to PayPal's iOS SDK. This library will help you accept card, PayPal, Venmo, and alternative payments in your iOS app.
```

Comment:
nit: can we update "alternative payments" => "alternative payment methods"? That appears to be the language we currently use in PayPal and Braintree docs for this.